jenkins/continuous-integration.jenkins

```diff
@@ -0,0 +1,41 @@
+pipeline {
+  agent {
+    docker { image 'node:10-slim' }
+  }
+  environment {
+    CODECOV_TOKEN = '523afe7a-4620-466f-a26a-d914c18a7c0a'
+  }
+  stages {
+    stage('Build') {
+      steps {
+        sh 'yarn'
+      }
+    }
+    stage('Test') {
+      parallel {
+        stage('Lint') {
+          steps {
+            sh 'yarn js-lint'
```

Comment:
What is the advantage of splitting the lint into multiple commands? I am afraid that people will forget to update this list when they add an additional linter (like `style-lint`, or `shellcheck`, or a JSON one, ...)

Reply:
i was on the fence about this, will combine them